packages/ui/IncomeExpenseIcon/index.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+import { Icon } from "ui";
+import { IconType } from "ui/Icon/index";
+
+type categoryType = {
+  id: number;
+  name: string;
+  icon: {
+    name: string;
+    foreground: string;
+    background: string;
+  };
+};
+
+export type IncomeExpenseIconProps = {
```

Comment:
you don't need `export` here

```suggestion
type IncomeExpenseIconProps = {
```

Reply:
I'm using this type in storybook, to pass prop `small` to parent wrapper :)

Reply:
ohh ok, I checked it as "viewed" and it wasn't found when I was searching where you used it :) nvm then!